.travis.yml

```diff
@@ -3,12 +3,18 @@ sudo: false
 cache:
   directories:
     - $HOME/.m2
-
 env:
   - JDK=oraclejdk8
 dist: trusty
 before_script:
   - jdk_switcher use $JDK
 
-script:
-  - mvn clean verify --batch-mode --errors --fail-at-end --show-version -DinstallAtEnd=true
+install: mvn clean verify -Dgpg.skip
+
+# TODO(weiguo) not tested yet
```

Comment:
In fact, it will fail to run the script `mvn ... ` due to the incorrect `gpg` configuration.
I left an issue #13